service/frontend/workflowHandler.go

```diff
@@ -3112,3 +3119,13 @@ func (wh *WorkflowHandler) bucketName(customBucketName *string) string {
 func (wh *WorkflowHandler) customBucketNameProvided(customBucketName *string) bool {
 	return customBucketName != nil && len(*customBucketName) != 0
 }
+
+func (wh *WorkflowHandler) getVisibilityManager(domain string) persistence.VisibilityManager {
```

Comment:
Can we wrap this under a wrapper implementation which also implements VisibilityMgr interface.

Reply:
good suggestion, updated.